app/uk/gov/hmrc/cataloguefrontend/shuttering/ShutterConnector.scala

```diff
@@ -88,4 +93,15 @@ class ShutterConnector @Inject()(
 
     http.PUT[ShutterStatus, Unit](s"$urlStates/$appName/${env.asString}", status)
   }
+
+
+  /**
+    * GET
+    * /shutter-api/outage-pages/{appName}/{environment}
+    * Retrieves the current shutter state for the given application in the given environment
+    */
+  def outagePageByAppAndEnv(appName: String, env: Environment)(implicit hc: HeaderCarrier): Future[Option[OutagePage]] = {
+    implicit val ssf = OutagePage.reads
+    http.GET[Option[OutagePage]](s"$urlOutagePages/$appName/${env.asString}")
```

Comment:
Are you sure this will give you a None if the outage page isn't there? Won't it fail the future with a 4xxException?

Reply:
Yup https://github.com/hmrc/http-verbs/blob/master/src/main/scala/uk/gov/hmrc/http/HttpReads.scala#L44